src/integration.test.ts

```diff
@@ -33,6 +37,25 @@ describe("Integration Test", (): void => {
     inMemoryCache = {};
     state = {};
 
+    let callCount = 0;
+
+    child_process.exec.mockImplementation(
+      (_command: any, _options: any, callback: any): any => {
+        let stdout;
+        if (
+          _command ==
```

Comment:
`==` --> `===` since [the former operator (loose equality) doesn't mean anything sensible in JavaScript](https://dorey.github.io/JavaScript-Equality-Table/).